global.d.ts

```diff
@@ -129,3 +130,16 @@ interface Document {
 // we are not using Chai anymore. We use Expect which has Jest style assertions
 // @TODO remove this once all test are refactored to use `epxect` instead of chai
 declare var expect: Chai.ExpectStatic;
+
+// TODO: Use proper types
+declare namespace JSX {
```

Comment:
Please try to discuss with @Hotell how to tackle this as I recall some issues he was mentioning...